kisanola/tech-community-frontend
# Setup git hooks

#### What does this PR do?
Setup git hook

#### Description of Task to be completed?
- [x] Setup git hook to run linting before committing
- [x] Fix linting issues

#### How should this be manually tested?
- Clone the repo
- Make some changes with eslint errors
- Try to commit to see your action rejected

#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
